Kwf/View/Helper/Money.php

```diff
@@ -16,6 +16,8 @@ public function setData(Kwf_Component_Data $data)
 
     public function money($amount)
     {
+        if ($amount === null || $amount === '') return '';
```

Comment:
why should value be empty?

Reply:
To prevent the following from throwing an exception if `checkSomething()` was not tested when returning `false`.  

**Component.php**
```php
public function getTemplateVars(Kwf_Component_Renderer_Abstract $renderer)
{
    $ret = parent::getTemplateVars($renderer);
    if ($this->checkSomething()) {
        $ret["price"] = 4.65;
    }
    return $ret;
}
```

**Component.twig**
```twig
<div>{{ price|money }}</div>
```

Reply:
return of checkSomething could be checked in template? render empty div should be prevented? alternative would be "-"
it makes no sense to support empty value when using money helper

Reply:
sollte, ja. Aber falls das vergessen worden ist crashed momentan alles - und genau das wollte ricky mit dem PR verhindern.

Reply:
warum er das macht ist mir schon klar ich frage mich halt nur muss der helper unterstützen das ihm nix übergeben wird?

Reply:
Ich finde schon, das wird in der Praxis sicher wieder mal vorkommen, dass irgendwer vergisst ein `if` zu machen... 

Jetzt wirft `number_format` einfach eine exception und im Frontend wird nur eine weiße Seite mit  `<h1>Error</h1>` ausgegeben... dass muss auf jeden fall (vor allem auf prod) vermieden werden.